packages/paste-core/components/textarea/src/index.tsx

```diff
@@ -1 +1,2 @@
-export * from "./TextArea";
+export { TextArea } from "./TextArea";
+export type { TextAreaProps, TextAreaVariants } from "./TextArea";
```

Comment:
question: why are you exporting TextAreaVariants

Reply:
Dunno, honestly